ext/node/ops/sqlite/statement.rs

```diff
@@ -369,6 +371,143 @@ impl StatementSync {
     Ok(arr)
   }
 
+  fn iterate<'a>(
+    &self,
+    scope: &mut v8::HandleScope<'a>,
+    #[varargs] params: Option<&v8::FunctionCallbackArguments>,
+  ) -> Result<v8::Local<'a, v8::Object>, SqliteError> {
+    macro_rules! v8_static_strings {
```

Comment:
We should just export this from `deno_core` - this is the third place we're redefining this macro